packages/common/src/components/UvText/UvText.test.tsx

```diff
@@ -82,6 +82,13 @@ describe('UvText', () => {
     expect(screen.getByText(/Ej angivet/i)).toBeInTheDocument()
   })
 
+  it('displaying text value if question is visible', () => {
```

Comment:
Since the logic that was implemented intended to hide elements, should we implement a test that verifies that an element is not showed if visibel is set to false?